subsys/suit/recovery_button/Kconfig

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config SUIT_RECOVERY_BUTTON
+	bool "Enable SUIT enter recovery button checking on startup"
+	depends on $(dt_chosen_enabled,recovery-button)
+	help
+	  This will make the firmware check if the recovery button specified in the
+	  device tree is pressed on startup. If it is, the firmware will enter
+	  the "foreground update mode", in which the same SUIT manifests are booted
+	  as in case of the recovery mode.
+
+	select SSF_SUIT_SERVICE_ENABLED
```

Comment:
help always goes last, select statements go above it

Reply:
Fixed